pkg/report/writer.go

```diff
@@ -41,7 +43,16 @@ func WriteResults(format string, output io.Writer, results Results, outputTempla
 	case "json":
 		writer = &JsonWriter{Output: output}
 	case "template":
-		tmpl, err := template.New("output template").Parse(outputTemplate)
+		tmpl, err := template.New("output template").Funcs(template.FuncMap{
+			"escapeString": func(input string) string {
```

Comment:
As I said, I think `escapeXML` is preferred because we might escape HTML or other formats other than XML.

Reply:
@knqyf263 Thanks, check now